src/core/Song.cpp

```diff
@@ -1043,7 +1043,7 @@ void Song::loadProject( const QString & fileName )
 			else
 			{
 				QTextStream(stderr) << tr("Can't load project: "
-					"Project file contains local paths to plugins.") << endl;
+					"Project file contains local paths to plugins.") << '\n';
```

Comment:
Let's keep `Qt::endl` because we build for Windows as well, which uses `\r\n` and Qt will account for that.

Reply:
Qt::endl doesn't build for most of the platforms according to the bot. Seems like I have to revert to just endl in that case.

Reply:
All 3 ways behave differently anyway, so yeah keeping it as it was is probably best for now until we clean more things up from core.
* `std::endl` will output proper OS newline
* `Qt::endl` will output proper OS newline AND flush
* '\n' is Unix/Unix-like, but as I understand it sometimes this can be translated to proper OS newline